src/constants/featureFlagsConstants.js

```diff
@@ -23,7 +23,9 @@ export const ENABLE_FEATURE_FLAG = 'ENABLE_FEATURE_FLAG';
 export const DISABLE_FEATURE_FLAG = 'DISABLE_FEATURE_FLAG';
 export const INITIAL_FEATURE_FLAGS = {
   SMART_WALLET_ENABLED: false,
+  BITCOIN_ENABLED: false,
 };
 export const DEVELOPMENT_FEATURE_FLAGS = {
   SMART_WALLET_ENABLED: true,
+  BITCOIN_ENABLED: false,
```

Comment:
This can be true if we want to check it in DEV

Reply:
I wasn't sure of how to handle this feature flag so I just added it as a placeholder back then. Now it's fixed :muscle: